src/AMQPRPCClient.js

```diff
@@ -42,13 +42,17 @@ class AMQPRPCClient extends AMQPEndpoint {
    * @example
    * client.sendCommand('some-command-name', [{foo: 'bar'}, [1, 2, 3]]);
    */
-  async sendCommand(command, args) {
+  async sendCommand(command, args, messageOptions) {
```

Comment:
I believe there is a reason for that. e.g. you want to have different message behaviour in the same channel\queue. So, you have known options ahead of connection establishment, like timeout, but you are not sure about specific message behaviour, like durability, persistent and so on.

thoughts ?

Reply:
### TL; DR:
 @jfromaniello, are you going to pass the option (```{ persistent: false }```) on each invokation of `sendMessage` or occasionally? 
If occasionally, I'd merge current implementation, otherwise It's worth to implement idea of @anton-kotenko .


### More verbose version of my opinion on this:
@anton-kotenko @jfromaniello I think both your approaches are could be implemented simultaneously here, they aren't muttual exclusive.

The suggestion of @anton-kotenko is about an ability to customize some message options for all messages, sent by the instance of the `AMQPRPCClient`.

The proposal of @jfromaniello is about an ability to customize message options per each command.

And we can implement both schemas –  to intoduce one more param in to the `constructor` (e.g. as `params.defaultMessageOptions`), which could be additionally customized by the `messageOptions` from `sendCommand` (which is introduced in this PR).

And accepting this PR won't affect the customization of default message options via `constuctor`, which we could implememt as soon as we need it (YAGNI).

Hence, I'd approve the PR.

Reply:
To be honest, I don't plan to change the properties for each message, but I agree it would be nice to have both. I just added `defaultMessageOptions` to the ctor